### PR TITLE
ContikiMac: Make it more configurable (Sub-GHz)

### PR DESCRIPTION
**Pull request by uknoblic**

### CODE REVIEW

**Comment by alignan** [comment]
Oops, the `.gitignore` was added again, you should exclude `.cc2538-cc1200-eth` as this is likely living only in your development box.

**Comment by uknoblic** [comment]
Antonio, how do i exclide .gitignore from this commit? Im close to giving up and setting up a complete new repo...

**Comment by alignan** [comment]
What I would do is:

`git rebase -i HEAD~2`

Then squash the commits (when prompted, replace ´pick´ with ´squash´ next to the `ContikiMac: Make it more configurable` commit message.  Then you would be prompted again to edit the commit message, delete the gitignore one and just leave the ContikiMac's.

Now you can edit the `.gitignore` file, add then do a `git commit --amend` and push again with `git push -f`

Maybe there's a better way but this is straightforward for me.

**Comment by g-oikonomou** [comment]
Or you can `git rebase master` and then in the editor you can delete or comment out the line showing the `.gitignore` commit.

File: core/net/mac/contikimac/contikimac.c [hunk 1]
```diff
@@ -106,7 +106,11 @@ static int we_are_receiving_burst = 0;
 
 /* INTER_PACKET_DEADLINE is the maximum time a receiver waits for the
    next packet of a burst when FRAME_PENDING is set. */
+#ifdef CONTIKIMAC_CONF_INTER_PACKET_DEADLINE
+#define INTER_PACKET_DEADLINE  		 		CONTIKIMAC_CONF_INTER_PACKET_DEADLINE
```
Comment (tsparber) [inline core/net/mac/contikimac/contikimac.c]: Maybe change these tabs to spaces (to match the other lines)? :)

Comment (g-oikonomou) [inline core/net/mac/contikimac/contikimac.c]: Yes, please use spaces

**Comment by alignan** [comment]
@uknoblic do you need a help with this? I can do the PR in your behalf if you like.

**Comment by uknoblic** [comment]
In fact this would be great! Even if i believe i would be able to handle these silly git issues, i would appreciate your help due to a heavy lack of time...

**Comment by alignan** [comment]
Closed via #1391

**Pull request closed without merging**